fala/apps/adviser/tests/test_search_view_function.py

```diff
@@ -41,7 +41,7 @@ def test_category_search_heading_closeness_and_matching(self):
 
     def test_search_returns_results_list(self):
         response = self.client.get(self.url, self.data)
-        soup = bs4.BeautifulSoup(response.content)
+        soup = bs4.BeautifulSoup(response.content, "html.parser")
```

Comment:
I'm interested in what this change does / is for as it's non-obvious to me. I might be missing something elsewhere in the PR

Reply:
It came up as a warning - it should have been in the original, but was missed